styles/error-page.scss

```diff
@@ -0,0 +1,56 @@
+@import './variables';
+
+.notfound {
+    position: relative;
+    background: $background-color ;
+    color: $font-primary-color;
+    font: $font-body;
+    height: 100vh;
+    text-align: center;
+    width: 100%;
+    display: inline-block;
+    
+    h1 {
+        font-size: 240px;
+        margin: 0px;
+        font-weight: 700;
+    }
+        
+    h2 {
+        font-size: 60px;
+        font-weight: 700;
+        margin: 0px 0px;
+    }
+
+    h3 {
+        font-size: 50px;
+        font-weight: 700;
+        margin-top: 0;
+    }
+    
+    p {
+        font-size: 14px;
+        font-weight: 400;
+        margin-bottom: 20px;
+
+    }
+    #error-btn {
+        font-size: 14px;
+        text-decoration: none;
+        text-transform: uppercase;
+        display: inline-block;
+        padding: 0px 10px;
+        border-radius: 40px;
+        font-weight: 700;
+        margin:0px;
+    }
+    
+}
+
+@include for-phone-only {
```

Comment:
in .notFound styling you can just

.notFound{
@include for-phone-only{
styles....
}
}

Also the name should be like, notFoundPage__container or smth like that, just notFound is not meaningfull enough

Reply:
I'll change that